src/main/java/cart/dao/CartProductDao.java

```diff
@@ -0,0 +1,15 @@
+package cart.dao;
+
+import cart.domain.cart.CartProduct;
+import cart.domain.member.Member;
+import cart.domain.product.Product;
+import java.util.List;
+
+public interface CartProductDao {
+
+    Long saveAndGetId(final CartProduct cartProduct);
+
+    List<Product> findAllProductByMember(final Member member);
+
+    int delete(final Long productId, final Member member);
```

Comment:
```suggestion
    int deleteByProductIdAndMember(final Long productId, final Member member);
```
이런식으로 의미 전달을 확실히 하는게 좋을수 있을거 같아요.